src/main/java/com/loofah/graph/api/services/GraphQLProvider.java

```diff
@@ -38,18 +34,22 @@
     private CategoryQuery categoryQuery;
     private CategoriesQuery categoriesQuery;
     private SkillsByCategoryQuery skillsByCategoryQuery;
+    private CraftsQuery craftsQuery;
+    private CraftQuery craftQuery;
 
     @Autowired
     public GraphQLProvider(SkillsQuery skillsQuery,
                            SkillQuery skillQuery,
                            CategoryQuery categoryQuery,
                            CategoriesQuery categoriesQuery,
-                           SkillsByCategoryQuery skillsByCategoryQuery){
+                           SkillsByCategoryQuery skillsByCategoryQuery, final CraftsQuery craftsQuery, final CraftQuery craftQuery){
```

Comment:
[tiny] could you put these on separate lines for consistency pls?

Reply:
Good spot, done now :)